tests/files/no-unused-modules/flow/flow-1.js

```diff
@@ -1,3 +1,3 @@
 // @flow strict
 export type Bar = number;
-export interface BarInterface {};
+export interface BarInterface {}
```

Comment:
why is this change needed? this should parse as an exported interface, *followed by* an empty statement, which should work fine.

Reply:
The other test files didn't have semicolons after interfaces/enums, so it was more a consistency/stylistic thing.

Reply:
In that case let's revert this; it's desirable for text fixtures to vary stylistically.

Reply:
I already reverted this; this is outdated.